### PR TITLE
Ki options #16

### LINKED ISSUE #16: Large Text
## User Story
As a user, with tired eyes, I should be able to chat with large text.

image
## AC
**Given** A user navigates to your page
**Then** they should see a checkbox below the input field called "Large Text"
**And** they should be able to click this checkbox
**And** when the checkbox is selected all the message text should be larger

## Dev Notes
- Click on the large text checkbox, that will update the value of the text that will pull from sass file where we stored the text size value.

### PR DESCRIPTION
**Pull request by Kamiran79**
This should close #16

### CODE REVIEW

File: src/javascripts/components/messageCards.js [hunk 1]
```diff
@@ -21,11 +20,11 @@ const messageBuilder = () => {
     domString += `
       <div class='messageCard'>
         <div class='messageTop d-flex'>
-            <div class='user'>${user}:</div>
+            <div class='user'>${user}</div>
```
Comment (JeremiahV93) [inline src/javascripts/components/messageCards.js]: add :

File: src/javascripts/components/messageCards.js [hunk 2]
```diff
@@ -21,11 +20,11 @@ const messageBuilder = () => {
...
             <div class='timestamp'>${arr[i].timestamp}</div>
         </div>
         <div class='messageBottom d-flex'>
-            <div class='message'>"${arr[i].message}"</div> 
+            <div class='message'>${arr[i].message}</div>
```
Comment (JeremiahV93) [inline src/javascripts/components/messageCards.js]: add ""s

**Review by JeremiahV93 — CHANGES_REQUESTED** [review]
add colon and quotes and it'll be good to go

**Review by Kamiran79 — COMMENTED** [review]
updated

**Review by JeremiahV93 — APPROVED** [review]
👍

**Pull request merged**